instrumentation/jetty/jetty-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jetty/common/JettyHandlerAdviceHelper.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jetty.common;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.instrumentation.servlet.ServletAccessor;
+import io.opentelemetry.instrumentation.servlet.ServletHttpServerTracer;
+import io.opentelemetry.instrumentation.servlet.TagSettingAsyncListener;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class JettyHandlerAdviceHelper {
+  /**
+   * Prepares the current context span for end. If the span ended exceptionally due to an exception,
+   * the span is ended within this method.
+   *
+   * <p>Returns <code>true</code> if the span can be ended normally. Returns <code>false</code> if
+   * scope or context is null (an existing span was found in start advice), or the span ended
+   * exceptionally, which is handled within this method.
+   */
```

Comment:
This is `void` method, it does not return anything.

Reply:
Fixed the doc (was out of date due to refactor).